docs/getting_started/configuration/poly_new/guide.rst

```diff
@@ -305,12 +305,12 @@ To enable ERT to read the simulation results, you'll need to use the :ref:`GEN_D
 
 .. code-block:: shell
 
-    GEN_DATA POLY_RES RESULT_FILE:poly_%d.out REPORT_STEPS:0 INPUT_FORMAT:ASCII
+    GEN_DATA POLY_RES RESULT_FILE:poly.out REPORT_STEPS:0 INPUT_FORMAT:ASCII
```

Comment:
Can we get rid of `REPORT STEPS` here?

Reply:
Yes, I got lost in rebasing, will fix it